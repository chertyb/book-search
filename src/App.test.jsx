import React from 'react';
import ReactDom from 'react-dom';
import App from './App';

describe('App', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(<App />, div);
    expect(div).toMatchSnapshot();
    ReactDom.unmountComponentAtNode(div);
  });
});
