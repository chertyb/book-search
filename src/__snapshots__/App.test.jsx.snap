// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div>
  <div
    class="App"
  >
    <div>
      <h1>
        Book Search App
      </h1>
      <input
        type="text"
        value=""
      />
      <button
        type="button"
      >
        Search
      </button>
      <div
        id="book-display"
      />
    </div>
  </div>
</div>
`;
